src/components/TokenSelect/CommonBases.tsx

```diff
@@ -0,0 +1,66 @@
+import { Currency } from '@uniswap/sdk-core'
+import Row from 'components/Row'
+import TokenImg from 'components/TokenImg'
+import { BASES_TO_CHECK_TRADES_AGAINST } from 'constants/routing'
+import { transparentize } from 'polished'
+import styled from 'styled-components/macro'
+import { ThemedText } from 'theme'
+import { currencyId } from 'utils/currencyId'
+
+const BasesContainer = styled(Row)`
+  margin: 0 1.25em;
+`
+
+const BaseWrapper = styled.div<{ active?: boolean }>`
+  align-items: center;
+  background-color: ${({ theme, active }) => (active ? transparentize(0.76, theme.active) : 'transparent')};
+  border: 1px solid ${({ theme, active }) => (active ? theme.active : theme.outline)};
+  border-radius: 1em;
+  color: ${({ theme, active }) => (active ? theme.active : theme.primary)};
+
+  display: flex;
+  :hover {
+    background-color: ${({ theme }) => transparentize(0.76, theme.active)};
+    color: ${({ theme }) => theme.active};
+    cursor: pointer;
+  }
+
+  padding: 0.5em 0.75em 0.5em;
+`
+
+const StyledTokenImg = styled(TokenImg)`
+  margin-right: 0.5em;
```

Comment:
Instead of modifying TokenImg can you add a gap in the BaseWrapper, so there are less bespoke elements?